### PR TITLE
Rename setting to enable tray icon

### PR DESCRIPTION
**Pull request by bwaldvogel**
"Show tray icon" instead of "Show icon on menu bar"

### CODE REVIEW

**Comment by ldziedziul** [comment]
:+1:

File: src/browser/settings.jsx [hunk 1]
```diff
@@ -168,7 +168,7 @@ var SettingsPage = React.createClass({
                      onChange={ this.handleChangeHideMenuBar } />);
     }
     if (process.platform === 'darwin' || process.platform === 'linux') {
-      options.push(<Input key="inputShowTrayIcon" id="inputShowTrayIcon" ref="showTrayIcon" type="checkbox" label="Show icon on menu bar (Need to restart the application)" checked={ this.state.showTrayIcon }
+      options.push(<Input key="inputShowTrayIcon" id="inputShowTrayIcon" ref="showTrayIcon" type="checkbox" label="Show tray icon (Need to restart the application)" checked={ this.state.showTrayIcon }
```
Comment (yuya-oc) [inline src/browser/settings.jsx]: Which platform are you considering? On macOS, I think "menu bar" is correct. Please see also #324.

**Comment by bwaldvogel** [comment]
Closing in favor of #324

**Comment by jasonblais** [comment]
Thanks @bwaldvogel for the initial pull request, hope to see you around!

**Pull request closed without merging**